echopype/tests/convert/test_convert.py

```diff
@@ -354,3 +443,168 @@ def test_convert_ek80(
         if export_engine == 'netcdf4' and output_save_path.startswith("s3://"):
             assert isinstance(e, ValueError) is True
             assert str(e) == 'Only local netcdf4 is supported.'
+
+
+def test_convert_es70(
+    es70_input_paths,
+    export_engine,
+    output_save_path,
+    minio_bucket,
+    model="ES70",
+):
+    common_storage_options = minio_bucket
+    output_storage_options = {}
+    ipath = es70_input_paths
+    if isinstance(es70_input_paths, list):
+        ipath = es70_input_paths[0]
+
+    input_storage_options = (
+        common_storage_options if ipath.startswith("s3://") else {}
+    )
+    if output_save_path and output_save_path.startswith("s3://"):
+        output_storage_options = common_storage_options
+
+    # Only using one file
+    ec = open_raw(
+        raw_file=ipath,
+        sonar_model=model,
+        storage_options=input_storage_options,
+    )
+
+    if (
+        export_engine == "netcdf4"
+        and output_save_path is not None
+        and output_save_path.startswith("s3://")
+    ):
+        return
+
+    if export_engine == "netcdf4":
+        to_file = getattr(ec, "to_netcdf")
+    elif export_engine == "zarr":
+        to_file = getattr(ec, "to_zarr")
+    else:
+        return
+    try:
+        to_file(
+            save_path=output_save_path,
+            overwrite=True,
+            output_storage_options=output_storage_options,
+        )
+
+        _check_output_files(
+            export_engine, ec.converted_raw_path, output_storage_options
+        )
+    except Exception as e:
+        if export_engine == 'netcdf4' and output_save_path.startswith("s3://"):
+            assert isinstance(e, ValueError) is True
+            assert str(e) == 'Only local netcdf4 is supported.'
+
+
+def test_convert_es80(
+    es80_input_paths,
+    export_engine,
+    output_save_path,
+    minio_bucket,
+    model="ES80",
+):
+    common_storage_options = minio_bucket
+    output_storage_options = {}
+    ipath = es80_input_paths
+    if isinstance(es80_input_paths, list):
+        ipath = es80_input_paths[0]
+
+    input_storage_options = (
+        common_storage_options if ipath.startswith("s3://") else {}
+    )
+    if output_save_path and output_save_path.startswith("s3://"):
+        output_storage_options = common_storage_options
+
+    # Only using one file
+    ec = open_raw(
```

Comment:
Change to `ec` to `echodata` so that it is consistent with other tests. Please change the other occurrences in this file also.